internal/sliceutils/sliceutils.go

```diff
@@ -34,10 +34,8 @@ func BulletedIndentedStringList(list []string) string {
 		return ""
 	}
 
-	result := make([]string, len(list))
-	for i, s := range list {
-		result[i] = s
-	}
+	var result = make([]string, len(list))
```

Comment:
mostly just asking so we don't have multiple patterns for the same thing - I think most of our code uses `result := append([]string{}, list...)` to make a copy of a slice - any reason for a difference here?

Reply:
It's a lint rule that was only triggered here. I think in most other situations we then modify the copied array. I think this is clearer about what's going on.